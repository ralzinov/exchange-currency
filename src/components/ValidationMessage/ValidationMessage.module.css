.host {
    font-size: 12px;
    color: red;
    line-height: 1.4;
    margin-top: 4px;
    height: 16px;
}
