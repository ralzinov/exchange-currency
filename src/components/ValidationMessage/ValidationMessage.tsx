import * as React from 'react';
import styles from './ValidationMessage.module.css';

export const ValidationMessage: React.FC = (props) => {
    return (
        <div className={styles.host}>{props.children}</div>
    );
};
