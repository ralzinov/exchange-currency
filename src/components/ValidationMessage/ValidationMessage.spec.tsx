import React from 'react';
import ReactDOM from 'react-dom';
import {ValidationMessage} from './ValidationMessage';
import {create} from 'react-test-renderer';

it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ValidationMessage>abc</ValidationMessage>, div);
    ReactDOM.unmountComponentAtNode(div);
});

it('should render with children', () => {
    const button = create(
        <ValidationMessage className="test">Label</ValidationMessage>
    );
    expect(button.toJSON()).toMatchSnapshot();
});

