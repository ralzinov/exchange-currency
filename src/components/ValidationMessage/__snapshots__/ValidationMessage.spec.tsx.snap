// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render with children 1`] = `
<div
  className="host test"
>
  Label
</div>
`;
